bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosXMLParser.java

```diff
@@ -1021,9 +1021,15 @@ public void characters(char @Nullable [] ch, int start, int length) throws SAXEx
      */
     public static String extractModelName(String sonosModelName) {
         String ret = sonosModelName;
+        // Remove until the first space (in practice, it removes the leading "Sonos " form the model name)
         Matcher matcher = Pattern.compile("\\s(.*)").matcher(ret);
         if (matcher.find()) {
             ret = matcher.group(1);
+            // Remove a potential ending text surrounded with parenthesis
+            matcher = Pattern.compile("(.*)\\s\\(.*\\)").matcher(ret);
+            if (matcher.find()) {
+                ret = matcher.group(1);
+            }
```

Comment:
This method might be good candidate for some unit test coverage which would clearly document how it works.

Reply:
Unit tests added